src/sentry/scripts/digests/digests.lua

```diff
@@ -9,22 +19,85 @@ function table.extend(t, items, length)
     end
 end
 
-function table.slice(t, start, stop)
-    if stop == nil then
-        stop = #t
+local function chunked(size, iterator, state, ...)
```

Comment:
This is a complicated function, here is proof that it works: https://gist.github.com/tkaemming/f063c2e7515e67d68ca37a08ba25e55c